proto/elys/incentive/apr.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+package elys.incentive;
+
+import "gogoproto/gogo.proto";
+import "cosmos_proto/cosmos.proto";
+
+option go_package = "github.com/elys-network/elys/x/incentive/types";
+
+message AprEdenElys {
+    string uusdc = 1 [
+        (cosmos_proto.scalar)  = "cosmos.Int",
+        (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+        (gogoproto.nullable)   = false
+    ];
+    string ueden = 2 [
+        (cosmos_proto.scalar)  = "cosmos.Int",
+        (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+        (gogoproto.nullable)   = false
+    ];
+    string uedenb = 3 [
+        (cosmos_proto.scalar)  = "cosmos.Int",
+        (gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Int",
+        (gogoproto.nullable)   = false
+    ];
+}
+
+message AprUsdc {
```

Comment:
Can you add a bit of comment on what this struct does and where it's used?

Reply:
@jelysn 
It is for tracking APR for each reward program.